.github/workflows/main.yml

```
@@ -97,5 +97,12 @@ jobs:
         if: ${{ always() }}
         run: moodle-plugin-ci behat --profile chrome
 
+      - name: Coverage
+        if: ${{ github.event_name != 'pull_request' && github.ref_name == 'main' }}
+        run: moodle-plugin-ci phpunit --coverage-clover
+
```

```suggestion
      - name: Comment test results into PR
         uses: EnricoMi/publish-unit-test-result-action@v1
         if: ${{ always() }}
         with:
           files: coverage.xml
```
Could the previous suggestion be done like this?